src/style.css

```diff
@@ -1 +1,38 @@
-/* add styles here */
+
+body,
+div,
+span {
+  margin: 0;
+  padding: 0;
+}
+
+
+.stars {
+  display: flex;
+  margin-bottom: 10px;
+}
+
+
+.stars__star {
+  width: 16px; 
+  height: 16px; 
+  margin-right: 4px; 
+  background-image: url('./images/star.svg'); 
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+
+
+.stars--0 .stars__star:nth-child(n) {
+  background-image: url('./images/star.svg'); 
+}
```

Comment:
we don't need that

Reply:
<img width="669" alt="image" src="https://github.com/user-attachments/assets/a6114aaa-1973-4504-acc5-88d18d832c85" />